sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxGatewayStoreModel.java

```diff
@@ -139,11 +139,6 @@ protected Map<String, String> getDefaultHeaders(
 
         defaultHeaders.put(HttpConstants.HttpHeaders.USER_AGENT, userAgentContainer.getUserAgent());
 
-        if (clientDefaultConsistencyLevel != null) {
```

Comment:
probably can remove the clientDefaultConsistencyLevel from the method parameters(does not see it used anywhere else in this method)